shared/src/main/scala/mlscript/JSBackend.scala

```diff
@@ -184,6 +184,151 @@ abstract class JSBackend(allowUnresolvedSymbols: Bool) {
     case _ => throw CodeGenError(s"ill-formed application $term")
   }
 
+  private def createASTCall(tp: Str, args: Ls[Term]): App =
+    App(Var(tp), Tup(args.map(a => N -> Fld(FldFlags.empty, a))))
+
+  case class FreeVars(vs: Set[Str])
```

Comment:
Make it a `class`. Case classes generate lots of bytecode which we have no use for here.